story1.html

```diff
@@ -1,18 +1,17 @@
 <!DOCTYPE html>
 <html>
-<head style= "background-color: #e5ba57">
+<head class= "story1Head">
```

Comment:
The `<head></head>` tag in HTML documents is not a visible element. Remove the class.